### PR TITLE
Disable form during certain times.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@TheoChevalier Turns out I didn't actually hit make pull request, thought I did...

Anyway, it's up now.

Do you want me to remove the string in the properties file?

**Comment by TheoChevalier** [comment]
@ScottDowne Yes please, or I can merge the other one in a few and I think GitHub will resolve automatically the conflict

**Comment by TheoChevalier** [comment]
Turns out git actually didn’t like the string conflict

**Comment by ScottDowne** [comment]
@TheoChevalier Fixed, mind taking a look?

File: src/components/call-tool.js [hunk 1]
```diff
@@ -96,7 +113,33 @@ module.exports = React.createClass({
...
+  renderClosedSign: function() {
+    var hoursString = "business_hours_cest";
+    if (this.isAfterOct29()) {
+      hoursString = "business_hours_cet";
+    }
+    return (
+      <section>
+        <h2>
+          {this.context.intl.formatMessage({id: 'closed_for_business'})}
+          <br/>
```
Comment (TheoChevalier) [inline src/components/call-tool.js]: in the other template we’re using `{" "}`, intended?

Comment (ScottDowne) [reply src/components/call-tool.js]: Yeah, in the other one we didn't want a line break, in this one it was requested to have a line break.

**Comment by TheoChevalier** [comment]
Just left a question, but nothing important. Looks good to me, I would say it’s good for staging then prod? Pomax’s patches and the date update are pending deployment, but I think that’s good to go too

File: src/components/call-tool.js (hunk 1 above)
Comment (TheoChevalier) [reply src/components/call-tool.js]: sounds good!

**Pull request merged**